Doc/library/pathlib.rst

```diff
@@ -339,7 +339,7 @@ Pure paths provide the following methods and properties:
    .. versionchanged:: 3.10
       The parents sequence now supports :term:`slices <slice>` and negative index values.
 
-.. data:: PurePath.parent
+.. attribute:: PurePath.parent
```

Comment:
There's a link at line 1274 that also needs to be updated.

Reply:
Does it?  References using mod, class, func, data, etc all look up their value in a unique namespace!

Reply:
I was thinking it's better to update for consistency and in case some tools now or in the future may indicate the type of link (or parse the link type and use it in some way). I agree the link still works and as far as our docs are concerned, it's a very minor nitpick and can be ignored.